src/main/java/org/maxgamer/rs/event/HandlerExecutor.java

```diff
@@ -1,77 +1,95 @@
 package org.maxgamer.rs.event;
 
 import java.lang.reflect.Method;
+import java.util.Arrays;
 
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://titan.maxgamer.org:9000/coding_rules#rule_key=squid%3AUndocumentedApi)